include/tfm/tfm_builtin_key_ids.h

```diff
@@ -7,6 +7,25 @@
 #ifndef __TFM_BUILTIN_KEY_IDS_H__
 #define __TFM_BUILTIN_KEY_IDS_H__
 
+#ifdef CONFIG_PSA_CRYPTO_DRIVER_CRACEN
+
+#define CRACEN_BUILTIN_IDENTITY_KEY_ID ((uint32_t)0x7fffc001)
+#define CRACEN_BUILTIN_MKEK_ID	       ((uint32_t)0x7fffc002)
+#define CRACEN_BUILTIN_MEXT_ID	       ((uint32_t)0x7fffc003)
```

Comment:
I see these are already defined by cracen_psa.h.

Why are we redefining them here instead of #including them from there?